.gitignore

```diff
@@ -0,0 +1,4 @@
+/target
+Cargo.lock
```

Comment:
We normally add the `Cargo.lock` file to the repo, to ensure that everyone who checks out the repo and builds it will end up with the same dependency versions, and make the upgrade to a newer dependency version explicit.